packages/connectivity/src/scp-cf/circuit-breaker.spec.ts

```diff
@@ -89,20 +87,17 @@ describe('circuit breaker', () => {
         await request();
         await sleep(50);
       } catch (e) {
-        if (
-          e ===
-          'Error in fetching the token for service my-xsuaa: Request failed with status code 400'
-        ) {
+        if (e.message.match(/Request failed with status code 500/)) {
           failedCalls++;
         }
-        if (e.message === 'Breaker is open') {
+        if (e.message.match(/Breaker is open/)) {
           keepCalling = false;
         }
       }
     }
     // Since we exit the loop breaker opened.
     expect(failedCalls).toBeGreaterThan(0);
-  }, 99999);
+  }, 999999);
```

Comment:
[req] reduce timeout

Reply:
I changed it for testing and forgot. Set it to 15 sec